<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1239</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Shopping Bot</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/Admin/Downloads/34649764Untitled-3-512.png</normaloff>C:/Users/Admin/Downloads/34649764Untitled-3-512.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>171</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(136, 138, 133);
font: 11pt &quot;Cantarell&quot;;</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Total Tasks</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbTotalTask">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>20</y>
          <width>21</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbIconTotalTask">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../bird_bot/images/tasks.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>30</y>
         <width>171</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(136, 138, 133);
font: 11pt &quot;Cantarell&quot;;</string>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Total Carts</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbTotalCarts">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>20</y>
          <width>21</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../bird_bot/images/cart.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>30</y>
         <width>171</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(136, 138, 133);
font: 11pt &quot;Cantarell&quot;;</string>
       </property>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Checkouts</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbTotalCheckouts">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>20</y>
          <width>21</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../bird_bot/images/success.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_4" native="true">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>30</y>
         <width>611</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(233, 185, 110);</string>
       </property>
       <widget class="QPushButton" name="btnNewTask">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>New Task</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnStartAll">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>10</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Start All</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnStopAll">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>10</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Stop All</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnDeleteAll">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>10</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete All</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>1201</width>
         <height>501</height>
        </rect>
       </property>
       <property name="title">
        <string>Tasks</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QTableWidget" name="tbListTask">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>true</bool>
          </property>
          <property name="columnCount">
           <number>5</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Site</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Product</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Account</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Status</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="btnStart">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>600</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnStop">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>600</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnEdit">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>600</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnDelete">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>600</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1239</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSetting"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>Manager</string>
    </property>
    <addaction name="actionAccount"/>
    <addaction name="actionBilling_Profile"/>
    <addaction name="actionProxies"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionCheck_for_update"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFIle"/>
   <addaction name="menuConfiguration"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionProfile_Manager">
   <property name="text">
    <string>Profile Manager</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCheck_for_update">
   <property name="text">
    <string>Check for update</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAccount">
   <property name="text">
    <string>Account</string>
   </property>
  </action>
  <action name="actionBilling_Profile">
   <property name="text">
    <string>Billing Profile</string>
   </property>
  </action>
  <action name="actionProxies">
   <property name="text">
    <string>Proxies</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="text">
    <string>Setting</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tbListTask</sender>
   <signal>itemDoubleClicked(QTableWidgetItem*)</signal>
   <receiver>btnEdit</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>619</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>689</x>
     <y>649</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
